pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -377,17 +351,16 @@ func TestValidateKubeProxyConfiguration(t *testing.T) {
 					TCPCloseWaitTimeout:   &metav1.Duration{Duration: 5 * time.Second},
 				},
 			},
-			msg: "only one CIDR allowed or a valid DualStack CIDR (e.g. 10.100.0.0/16,fde4:8dba:82e1::/48)",
+			expectedErrs: field.ErrorList{field.Invalid(newPath.Child("ClusterCIDR"), "192.168.59.0/24,fd00:192:168::/64,a.b.c.d/f", "only one CIDR allowed or a valid DualStack CIDR (e.g. 10.100.0.0/16,fde4:8dba:82e1::/48)")},
```

Comment:
The test name suggests that this is testing the bad CIDR string, but the error suggests it's failing because of the _number_ of CIDR strings, not because one of them doesn't parse...

Reply:
This is true, I'm uncertain how to rephrase this. This is probably another instance where the TODO comment I intend to fix above would help with.